### PR TITLE
add margin bottom for iPhone X

### PR DESCRIPTION
**Pull request by alimek**
because `SafeViewArea` doesn't work in modal, we must add margin bottom for button.

### CODE REVIEW

**Comment by alimek** [comment]
fix for #146

File: src/CustomDatePickerIOS/index.style.js [hunk 1]
```diff
@@ -1,4 +1,10 @@
-import { StyleSheet } from "react-native";
+import { StyleSheet, Dimensions, Platform } from "react-native";
+
+export const isIphoneX = () => {
+  const { height, width } = Dimensions.get("window");
+
+  return Platform.OS === "ios" && (height === 812 || width === 812);
+};
```
Comment (mmazzarolo) [inline src/CustomDatePickerIOS/index.style.js]: Could you update the check to this?
```javascript
return (
        Platform.OS === 'ios' &&
        !Platform.isPad &&
        !Platform.isTVOS &&
        (height === 812 || width === 812)
    );
```

**Comment by mmazzarolo** [comment]
Thank you @alimek , one minor think and then I'll merge it!

File: src/CustomDatePickerIOS/index.style.js (hunk 1 above)
Comment (alimek) [reply src/CustomDatePickerIOS/index.style.js]: fixed

**Comment by alimek** [comment]
i added also one fix more.

`onDateChange` was not executed from `CustomDatePickerIOS` component, because default props was
```
onDateChange: Function.propotype
```

and it was overwriting props in render function.

**Comment by mmazzarolo** [comment]
Merged, thanks!

**Comment by alimek** [comment]
@mmazzarolo thanks, can i ask for NPM release  ? :) 👍

**Comment by mmazzarolo** [comment]
[Sure, you can](https://github.com/mmazzarolo/react-native-modal-datetime-picker/releases/tag/v5.1.0)!

**Pull request merged**